### ISSUE TITLE
TeamSkeet X series change

### ISSUE THREAD

**Issue by covrdinbeez** [PAhelper/PhoenixAdult.bundle#1234]
**Describe the bug**
Looks like the series "TeamSkeet X Fucking Awesome" has been changed and a space added, it's now returning as "Team Skeet X Fucking Awesome." Previously matched scenes are under the collection "TeamSkeet X Fucking Awesome" and all of the other "TeamSkeet X" series don't have a space in between Team and Skeet.

https://www.teamskeet.com/series/fucking-awesome

**Include**
1. Scene URL
2. File Name
3. Log File

**Comment by DirtyRacer1337** [comment]
And what? Spaces not important

**Comment by DirtyRacer1337** [comment]
I didn't notice that part about collections. I thought you mean search.

You can select all movies with old collection name and edit it all at once with adding new collection name and delete after that old one.

**Comment by covrdinbeez** [comment]
So I dug a little more into this and maybe this is an issue with whether a site is specifically listed or not? I looked into PAsiteList.py and find no listing for TeamSkeet X Fucking Awesome even though a bunch of the other TeamSkeet X series are listed. This specific one isn't.

I tried a bit of manual searching on my own, and if I manually search for "TeamSkeet Extra Homework" it returns the series as "Team Skeet X Fucking Awesome" but if I manually search for "TeamSkeetXFuckingAwesome Extra Homework" it returns the series as "TeamSkeet X Fucking Awesome."

I guess I'm now confused as to why one is returning a space and another is not.

Maybe it's time for me to learn github and start digging into this stuff myself. ;)
